README.md

```diff
@@ -6,7 +6,7 @@ Redirect uri must be **http://<edx_url>/auth/complete/custom-oauth2/**
 
  - Install this client
    ```
-   pip install git+https://github.com/raccoongang/edx-oauth-client.git@hawthorn-master#egg=edx_oauth_client
+   pip install git+https://github.com/raccoongang/edx-oauth-client.git@ucdc-hawthorn-master#egg=edx_oauth_client
```

Comment:
Do we propose installing the client from the branch directly? I would prefer to install release pinned to the tag.

Reply:
Good point thnx. Pushed a fix.